manifests/params.pp

```diff
@@ -45,14 +45,8 @@
       ]
     }
     'LinuxMint': {
-      case fact('lsbmajdistrelease') {
-        # Linux Mint 18* is based on Ubuntu 16.04
-        default: {
-          $origins            = [
-            '${distro_id}:${distro_codename}-security', #lint:ignore:single_quote_string_with_variables
-          ]
-        }
-      }
+      # Linux Mint 18* is based on Ubuntu 16.04
```

Comment:
Is this comment still relevant?

Reply:
mhm probably not, since we don't have anything mint 18 specific. I deleted it.